charts/thoras/tests/secrets_collector_test.yaml

```diff
@@ -8,20 +8,20 @@ tests:
       slackWebhookUrlSecretRefKey: "url"
     asserts:
       - equal:
-          path: spec.template.spec.containers[1].env[2].valueFrom.secretKeyRef.name
+          path: $..spec.containers[1].env[?(@.name == 'SLACK_WEBHOOK_URL')].valueFrom.secretKeyRef.name
           value: "frou-frou"
       - equal:
-          path: spec.template.spec.containers[1].env[2].valueFrom.secretKeyRef.key
+          path: $..spec.containers[1].env[?(@.name == 'SLACK_WEBHOOK_URL')].valueFrom.secretKeyRef.key
           value: "url"
```

Comment:
how would these paths both point to the same value?

Reply:
the paths look the same to me - maybe Im interpreting the syntax wrong?

Reply:
I might not be understanding the question, but the first is comparing the `.name` value, and the second is looking at the `.key`. Am I misunderstanding your question?

Reply:
My window was small and I didnt see the `.key` and `.name` at the end...my bad!